graylog2-web-interface/src/hooks/useScopePermissions.ts

```diff
@@ -50,7 +50,7 @@ const useGetPermissionsByScope = (entity: Partial<GenericEntityType>) => {
     },
   );
 
-  const scope = entity._scope ? entity._scope.toUpperCase() : 'DEFAULT';
+  const scope = entity?._scope.toUpperCase() || 'DEFAULT';
```

Comment:
I'm not entirely sure, but I think existing entities would not have a `_scope` field. If that's the case, this will throw an error when trying to `toUpperCase` an `undefined` variable.

Reply:
Actually this is the reason why I made the changes to this line. Before it would throw an error when trying to access `_scope`, but using the optional chaining it would result in `undefined` if the `_scope` doesn't exist, and therefore will return 'DEFAULT'

Reply:
So the actual error was assuming the entity will always be there to initialize the hook.

This works because the evaluation stops at the entity value. This is undefined, so the rest isn't executed, returning the `'DEFAULT'` value that we want.

Reply:
Create a PR https://github.com/Graylog2/graylog2-server/pull/13365 to fix https://github.com/Graylog2/graylog2-server/issues/13361